tests/server_test.go

```diff
@@ -9857,6 +9858,39 @@ func TestServer_Prometheus_Write(t *testing.T) {
 	}
 }
 
+func TestCliEndToEnd(t *testing.T) {
```

Comment:
Can we add a test that returns more than one row? `cmd/influxd/launcher/query_test.go` has some example e2e tests that could be adapted.